config/core/test-configs.yaml

```diff
@@ -2025,6 +2025,13 @@ test_configs:
   - device_type: imx6q-sabrelite
     test_plans:
       - baseline
+      - kselftest-filesystems
+      - kselftest-futex
+      - kselftest-lib
+      - kselftest-livepatch
+      - kselftest-lkdtm
```

Comment:
Failed: https://staging.kernelci.org/test/plan/id/623379d4df4661d6d44b31a0/

Reply:
Logs on this link aren't found. In the last couple of weeks, this test hasn't run on this target again. So we aren't sure if this is a problem or not.

Reply:
It looks like a genuine kernel problem: https://lava.collabora.co.uk/scheduler/job/6019351
So I think it's OK to enable it, and the problem should be fixed in the kernel.